templates/artist/artist_list.html

```diff
@@ -100,7 +100,7 @@ <h2 class="artist-list-overlay banner-funded">FUNDED</h2>
                              {% elif latest_campaign.open %}
                                  <h2 class="artist-list-overlay banner-active">
                                     {% if latest_campaign.percentage_funded %}
-                                        {{ latest_campaign.percentage_funded }}% FUNDED
+                                        ACTIVE
                                     {% elif campaign_status == 'Recent' %}
                                         ACTIVE
```

Comment:
Hey @localastronaut, after looking at this code a little bit more closely I just realized that it's never possible for the campaign status to appear as "Recent" and that's been true since we reintroduced the campaign banner 3 months ago. The `campaign_status` was originally part of the feature to filter by campaign status, which we've removed, so it doesn't mean anything anymore.

Anyways, I think this code should be refactored to reflect how it's currently working. Before I do that, I just want to check to make sure I understand how it is that you want it to work:

1. You want the banner to show `COMING SOON` for artists that have never had a campaign
1. You want the banner to show `FUNDED` for artists that successfully funded their most recent campaign
1. You want the banner to show `ACTIVE` for artists with a campaign that's currently running
1. You do not want any banner to show under any other circumstance

Is that right? If so, this is currently incorrect as implemented because it will only show `ACTIVE` if the campaign currently running has at been funded to at least 1%. I can fix that in my refactor though.

Reply:
Yes, that is correct!